modules/aerospike/aerospike.go

```diff
@@ -0,0 +1,71 @@
+package aerospike
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	port          = "3000/tcp"
+	containerName = "tc_dynamodb_local"
+)
+
+type AerospikeContainer struct {
+	testcontainers.Container
+	Host string
+	Port int
```

Comment:
bug: let's make these fields private to avoid users changing the values. Let's create accessor methods to the container in case we need to retrieve them